test/units/testsuite-29.sh

```diff
@@ -116,11 +118,16 @@ portablectl detach --now --enable --runtime /tmp/minimal_1 minimal-app0
 portablectl list | grep -q -F "No images."
 busctl tree org.freedesktop.portable1 --no-pager | grep -q -F '/org/freedesktop/portable1/image/minimal_5f1' && exit 1
 
+# Need to figure out why this is not working
 portablectl "${ARGS[@]}" attach --now --runtime --extension /usr/share/app0.raw /usr/share/minimal_0.raw app0
+portablectl "${ARGS[@]}" attach --now --runtime --extension /etc/app00.raw /etc/minimal_00.raw app00
```

Comment:
fold into the above, the purpose of confext is to provide configuration files, not new services